dashboard/src/components/Accordion/Accordion.tsx

```diff
@@ -165,7 +167,14 @@ const AccordionBuildsTrigger = ({
     <>
       <TableCell>{triggerInfo.config}</TableCell>
       <TableCell>{triggerInfo.compiler}</TableCell>
-      <TableCell>{triggerInfo.date}</TableCell>
+      <TableCell>
+        <TooltipDateTime
+          dateTime={triggerInfo.date || ''}
```

Comment:
it shouldn't pass an empty string if there is no date
it should just not render the component
otherwise, it will get an Invalid Date